app/.storybook/preview.tsx

```diff
@@ -35,7 +36,13 @@ const parameters = {
   },
 };
 
+const i18nMessagesLoader: Loader = async (context) => {
+  const messages = await getMessagesWithFallbacks(context.globals.locale);
+  return { messages };
+};
+
 const preview: Preview = {
+  loaders: [i18nMessagesLoader],
```

Comment:
This puts `messages` on `context.loaded.messages`, which is available to decorators and stories